### PR TITLE
Ncv3 support

### PR DESCRIPTION
**Pull request by rafsalas19**
- Adding NCv3 support

### CODE REVIEW

File: conf/nc6s_v3.conf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+###
+# ncv3nhcvm || export MARK_OFFLINE=1 NHC_CHECK_ALL=0
+
+
+#######################################################################
+###
+### Hardware checks
+###
+ * || check_hw_cpuinfo 1 6 6
+ * || check_hw_physmem 110GB 110GB 3%
```
Comment (jeseszhang1010) [inline conf/nc6s_v3.conf:23]: NIT: Some SKUs are with kB, some are with MB, and GB.
Do we want unify the unit?

File: run-health-checks.sh [hunk 2]
```diff
@@ -93,34 +93,42 @@ if [ -z "$CONF_FILE" ]; then
         conf_name="hb120rs_v3"
     elif echo "$SKU" | grep -q "hb176rs_v4"; then
         conf_name="hb176rs_v4"
-	an_rate=100
+	    an_rate=100
```
Comment (jeseszhang1010) [inline run-health-checks.sh]: Indentation off?

**Review by jeseszhang1010 — APPROVED** [review]

File: run-health-checks.sh (hunk 2 above)
Comment (rafsalas19) [reply run-health-checks.sh]: Yea, it shows different in different terminals/editors. Let me revert as I thought I was actually fixing the indention issue

File: conf/nc6s_v3.conf (hunk 1 above)
Comment (rafsalas19) [reply conf/nc6s_v3.conf:23]: Yea that's a bit weird. This check was auto generated by NHC. Let me think about this one, I will probably not address in this PR.

File: run-health-checks.sh (hunk 2 above)
Comment (rafsalas19) [reply run-health-checks.sh]: addressed

**Pull request merged**